### ISSUE TITLE
Flat spectra below 0.5 eV?

### ISSUE THREAD

**Issue by kpolsen** [SKIRT/SKIRT9#72]
**Description**
I need help to understand the spectral shape I'm seeing per cell. In the FUV range I see an expected range in dust attenuation, as seen in this plot where all cells of a galaxy are binned in terms of their NIR-to-FUV flux ratio:

![skirt_spectra_G146](https://user-images.githubusercontent.com/13789517/88407970-c5bd0680-cd87-11ea-8540-166160e6d738.png)

But is the flat spectra below about 0.5 eV to be expected? Is there a way to plot the raw SED tables that SKIRT is using?

I use the Bruzual Charlot SED family with Chabrier IMF:

```
                        <sedFamily type="SEDFamily">
                            <BruzualCharlotSEDFamily imf="Chabrier" resolution="Low"/>
                        </sedFamily>
                        <wavelengthBiasDistribution type="WavelengthDistribution">
                            <LogWavelengthDistribution minWavelength="0.0001 micron" maxWavelength="1e6 micron"/>
                        </wavelengthBiasDistribution>
```

**Data**
I can supply ski file and data if needed.

**System**
Linux, Ubuntu, SKIRT v9.

**Context**
Ideally, I would like to pass the spectra per cell from SKIRT directly into Cloudy models that calculate line emission. However, Cloudy requires the full spectrum from ~30 m to 1.2e-14 m in wavelength range, and I don't think SKIRT will be reliable on that range because it doesn't account for extinction by e.g. Hydrogen. Hence the approach above, where I calculate the NIR-FUV ratio and match that in incoming spectrum to Cloudy.

**Comment by petercamps** [comment]
The stellar emission for wavelengths longer than 2.5 micron (corresponding to a photon energy below 0.5 eV) is fairly low and does not have many spectral features. That's probably why you get the results shown. 

You can plot the raw SEDs used by SKIRT through the [PTS command line](https://skirt.ugent.be/pts9/namespacepts_1_1visual_1_1do_1_1plot__stored__table.html#details) or through the [PTS plotStoredTableInteractive function](https://skirt.ugent.be/pts9/namespacepts_1_1visual_1_1plotstoredtable.html) used from a [notebook](https://skirt.ugent.be/pts9/_user_notebooks.html).

Alternatively, you can make a SKIRT ski file that imports just a single source particle with the relevant properties, has no media, and records the spectrum using an SED instrument. The benefit of this approach is that it also includes any interpolation of the tabulated spectra done inside SKIRT; also you don't need to install and learn PTS.

I hope this helps.

**Comment by kpolsen** [comment]
I went ahead with your suggestion to run a single one-source-particle scenario using a 1e4 Msun star particle of age 1 Myr and solar metallicity to sort of get an lower limit on the NIR-FUV flux ratio. 

This is the resulting SED, this time as function of wavelength to see the emission above 2.5 microns (in the 2nd plot I integrated over wavelengths to compare with the previous figure):

![image](https://user-images.githubusercontent.com/13789517/89811092-fc6a7f00-daf2-11ea-920a-8b9a6e8b51ed.png)

If these plots don't look too surprising to you, I will probably use this as the base case and compare cells with attenuated spectra. The spectra in Cloudy, even with no dust, do not become flat like this at E < 0.5 eV, but maybe I can look at relative attenuation compared to the no-dust scenario in order to link SKIRT and Cloudy.
